File: OneToOneCallSample/app/src/main/java/com/stringee/apptoappcallsample/stringee/common/CallStatus.java
package com.stringee.apptoappcallsample.stringee.common;

/** Stable call states exposed to the host UI. */
public enum CallStatus {
    INCOMING("Incoming"),
    CALLING("Calling"),
    RINGING("Ringing"),
    STARTING("Starting"),
    STARTED("Started"),
    RECONNECTING("Reconnecting"),
    BUSY("Busy"),
    ENDED("Ended");

    private final String value;

    CallStatus(String value) {
        this.value = value;
    }

    public String getValue() {
        return value;
    }
}

File: Kotlin-OneToOneCallSample/app/src/main/java/com/stringee/kotlin_onetoonecallsample/stringee/common/CallStatus.kt
package com.stringee.kotlin_onetoonecallsample.stringee.common

/** Stable call states exposed to the host UI. */
enum class CallStatus(value: String) {
    INCOMING("Incoming"),
    CALLING("Calling"),
    RINGING("Ringing"),
    STARTING("Starting"),
    STARTED("Started"),
    RECONNECTING("Reconnecting"),
    BUSY("Busy"),
    ENDED("Ended");

    val value: String

    init {
        this.value = value
    }
}
